### ISSUE TITLE
[Feature Request] Add HTMLElement support for editArea parameter.

### ISSUE THREAD

**Issue by woody-li** [paulrosen/abcjs#1001]
In interactive Editor Constructor, the editArea parameter only supports ID string and EditArea instance.

Is it possibile to add HTMLElement type support, I saw that other id-like parameters both supports it.

In my case, the element is descendant of a shadowDom, so it cann't be find by getElementById.

**Comment by paulrosen** [comment]
So you want to do something like:
```
const editArea = new abcjs.EditArea(element)
const editor = new abcjs.Editor(editArea)
```
That wouldn't be hard to do. I'll have it in the next version.

**Comment by paulrosen** [comment]
This will be fixed in version 6.4.0

**Comment by woody-li** [comment]
It works fine, thanks
